source/blog/2018-12-21-the-ember-times-issue-78.md

```diff
@@ -61,8 +60,24 @@ Read and comment on the [full RFC on GitHub](https://github.com/emberjs/rfcs/pul
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [EmberConf Registration is Open! 💁‍♀️](https://emberconf.com/index.html)
+
+2019 is rapidly approaching, and you've got one more thing to do before EOY: buy your EmberConf ticket! This year's Early-Bird discount is significant ($50), while supplies last or until 12/31. So before anything else, here's that big shiny button! 🆗
+
+<div class="blog-row">
+  <a class="ember-button" style="width:260px" href="https://emberconf.com/register.html">Register Now</a>
+</div>
+
+The program has been announced and is looking really great: so many new faces, and a bunch of familiar old ones! Be sure to check out the [full schedule](https://emberconf.com/schedule.html) for all the details. 
 
+We're particularly excited about [Inclusiveness at EmberConf](https://emberconf.com/inclusiveness-at-emberconf.html) this year. In an industry lacking attendee and speaker diversity, EmberConf strives to be an exception. Highlights of this year's efforts include:
+ 
+ <!--alex ignore gals-men-->
```

Comment:
Woops, it seems several messages need to be ignored like so:

```suggestion
  <!--alex ignore gals-men kids-->
```

Reply:
fixed it `<!--alex ignore gals-men women kids-->`